### ISSUE TITLE
Payload undefined instead of unset

### ISSUE THREAD

**Issue by sebastiancarlsson** [acdlite/redux-actions#44]
Hi, I just started toying around with this lib so I'm probably missing something important. The [spec for FSA](https://github.com/acdlite/flux-standard-action#actions) states that an action MAY have a `payload` property, however when I do this:

```javascript
const type = 'TYPE';
const myActionCreator = createAction(type);
const action = myActionCreator();
console.log(action); // <- {type: "TYPE", payload: undefined} but I expected {type: "TYPE"}
```

Is there a reason for this?

**Comment by sebastiancarlsson** [comment]
EDIT: My use case is that I don't want to have to add "payload:undefined" to all my tests, i.e. this will fail:
```javascript
const expectedAction = { type: 'TYPE' };
expect(myActionCreator()).to.deep.equal(expectedAction);
```

**Comment by john-mortlock** [comment]
I am having the same issue, so I created a pull request to fix this.

**Comment by timche** [comment]
Could be fixed with #48.

**Comment by yangmillstheory** [comment]
Looks like #48 fixes this (with tests), can we close?

**Comment by yangmillstheory** [comment]
@timche Isn't this closed in 0.10.0?

**Comment by timche** [comment]
You are right @yangmillstheory, thanks for the hint :)
